conduct.md

```diff
@@ -120,15 +120,21 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness.
+    It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open can also allow your team members to help you with problems you did not catch or understand withing your code.
```

Comment:
Thanks for your hard work. Based on the issue tracker of the main file ( the one which is not in our fork), the subject is better to be "students" other than "you". Please consider to use "you"